lisa/tools/rc_local.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PurePosixPath
+
+from lisa.executable import Tool
+from lisa.tools.echo import Echo
+
+
+class RcLocal(Tool):
```

Comment:
Because it's a virtual tool. Add some comments to explain what it uses.

Reply:
But I'm not sure if there is better name. The rc.local is a file name, it's used to save startup scripts. The tool name can be related to the task instead of file name.